lib/vmdb/initializer.rb

```diff
@@ -12,8 +12,10 @@ def self.init
       #   * command line(rails server)
       #   * debugger
       if defined?(Rails::Server)
-        MiqUiWorker.preload_for_worker_role
+        # preload_for_worker_role depends on seeding, principally MiqDatabase
```

Comment:
Yes, see #7402... sorry, I had these two PRs together at first but the diff looked confusing so I split them up since they're completely independent.